js/__tests__/Heap.spec.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { NativeModules } from 'react-native';
+
+import Heap from '../Heap';
+
+describe('The Heap object', () => {
```

Comment:
It'd be nice if there were some tests for the happy path with more than an empty object.

Reply:
Oh, duh... nice catch... will add and merge